test/oc_trace_context_SUITE.erl

```diff
@@ -44,3 +44,13 @@ decode_with_extra_junk(_Config) ->
     Encoded = oc_trace_context_binary:encode(Decoded),
 
     ?assertMatch(Binary, Encoded).
+
+encode_decode_headers(_Config) ->
+    %% TraceId: 4bf92f3577b34da6a3ce929d0e0e4736
+    %% SpanId: 00f067aa0ba902b7
+    %% Enabled: true
+    Header = <<"00-4bf92f3577b34da6a3ce929d0e0e4736-00f067aa0ba902b7-01">>,
+    Decoded = oc_trace_context_headers:decode(Header),
+    Encoded = oc_trace_context_headers:encode(Decoded),
+    ?assertEqual(Header, list_to_binary(Encoded)),
+    ?assertEqual(Decoded, oc_trace_context_headers:decode(Encoded)).
```

Comment:
<super nitpicky mode>
* disabled isn't ever tested
* hanlding of invalid span id's isn't tasted
</super nitpicky mode>